cluster/examples/kubernetes/ceph/csi/example/rbd/configure-storageclass.bash

```diff
@@ -0,0 +1,105 @@
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+EOF
+
+# Ensure that we have a kubernetes login configured on the ceph cluster
+(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph -c /var/lib/rook/rook-ceph/rook-ceph.config auth get-or-create-key client.kubernetes mon \"allow profile rbd\" osd \"profile rbd pool=rbd\"")
+
+# Retrieve the base64 encoded passwords for the admin and kubernetes users on the cluster and store them in env variables to use for storageclass config
+admin_key=$(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.admin -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
+kubernetes_key=$(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.kubernetes -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
+
+# Use the keys from above to create a secrets object on our Kubernetes cluster
+<< EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: csi-rbd-secret
+  namespace: default
+data:
+  # Key value corresponds to a user name defined in Ceph cluster
+  admin: ${admin_key}
+  # Key value corresponds to a user name defined in Ceph cluster
+  kubernetes: ${kubernetes_key}
+  # if monValueFromSecret is set to "monitors", uncomment the
+  # following and set the mon there
+  #monitors: BASE64-ENCODED-Comma-Delimited-Mons
+EOF
+
+# Gather the Cluster IPs of our mons (we assume the default port here)
+OUT=$(kubectl get service -n rook-ceph | awk '/rook-ceph-mon/ { print $3;  }')
+IFS='
+'
+count=0
+for i in $OUT
+do
+        if [ $count == 0 ]; then
+                MONA=$i
+        elif [ $count  == 1 ]; then
+                MONB=$i
+        elif [ $count == 2 ]; then
+                MONC=$i
+        else
+                echo "I only expect 3 mons for now"
+        fi
+        let count=count+1
+done
+
+#Create our csi-rbd storage class
+<< EOF | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: csi-rbd
+provisioner: rbd.csi.ceph.com
+parameters:
+    # Comma separated list of Ceph monitors
+    # if using FQDN, make sure csi plugin's dns policy is appropriate.
+    monitors: $MONA:6789,$MONB:6789,$MONC:6789  
+
+    # if "monitors" parameter is not set, driver to get monitors from same
+    # secret as admin/user credentials. "monValueFromSecret" provides the
+    # key in the secret whose value is the mons
+    #monValueFromSecret: "monitors"
+    
+    # Ceph pool into which the RBD image shall be created
+    pool: rbd
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
+
+    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
```

Comment:
`layering` -> layering

Reply:
That's actually an existing line (just moved); I can change it of course; but honestly I'm concerned there will then be another review pointing out that it shouldn't be changed.

Reply:
it is throwing an error when I run the script because of "`" around the word layering.